### PR TITLE
 Automated qr code generation

### PR DESCRIPTION
**Pull request by adityaSINGHrox**
This is a simple real world application of python. I have tested this code for 950 entries at a time. I know the readme file isn't that descriptive plz tell if u need any changes.

### CODE REVIEW

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# QR Code Generator
+This is a simple code that can generate qr codes for various purposes. Few use cases are explained below.
+
+## Event Invitations
+Ever had to organize hackathons/workshops? Some  hackathons are 24 hours long, the organizers arrange for food goodie etc..
+How to  ensure everyone get exactly what the y paid for. Sure we can use old systems like tokens, tickets etc. but the tend to use too much of human resource and also waste paper.
```
Comment (Rishit-dagli) [inline Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:6]: A small typo, "you"

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# QR Code Generator
+This is a simple code that can generate qr codes for various purposes. Few use cases are explained below.
+
+## Event Invitations
+Ever had to organize hackathons/workshops? Some  hackathons are 24 hours long, the organizers arrange for food goodie etc..
```
Comment (Rishit-dagli) [inline Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:5]: A small typo, you have included, double full stops

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# QR Code Generator
```
Comment (Rishit-dagli) [inline Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:1]: Could you add this just under the heading?

```
_Originally contributed by [@adityaSINGHrox](https://github.com/adityaSINGHrox)_
```

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]
The #12 mentions the uses for each of the folders.
Instead, of having this in the Training Modules folder could have these files under `Projects/QR Code Generator` ?

**Comment by Rishit-dagli** [comment]
@adityaSINGHrox I also see you haven't agreed to the CLA mentioned in [this comment](https://github.com/Rishit-dagli/Python3-by-practice/pull/26#issuecomment-702179383), could you sign it, so I could go forward with these changes?

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md (hunk 3 above)
Comment (adityaSINGHrox) [reply Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:1]: i'll do that

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md (hunk 2 above)
Comment (adityaSINGHrox) [reply Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:5]: i thought sentences ending with etc have double full stops 
as in 1 for etc and one normal full stop 
no worries i'll change it

Comment (Rishit-dagli) [reply Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:5]: @adityaSINGHrox Oh, I use Grammarly for checking my READMEs, if this seems good to you and is accepted, I have no problem with it

**Comment by adityaSINGHrox** [comment]
> The #12 mentions the uses for each of the folders.
> Instead, of having this in the Training Modules folder could have these files under `Projects/QR Code Generator` ?

i'll do that too

File: Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md (hunk 2 above)
Comment (adityaSINGHrox) [reply Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md:5]: > @adityaSINGHrox Oh, I use Grammarly for checking my READMEs, if this seems good to you and is accepted, I have no problem with it

oh if that's the case i'll do it

**Comment by adityaSINGHrox** [comment]
@Rishit-dagli I Just noticed that if someone wanted to test the code 
they must some data/csv to check it 
what's ur take on it

**Comment by Rishit-dagli** [comment]
> @Rishit-dagli I Just noticed that if someone wanted to test the code
> they must have some data/csv to check it
> what's ur take on it

Yeah, I had earlier noticed you use a `.csv` file to ingest the data. But since you brought this up, would you like to add a sample `.csv` file, as an example, if that is what you meant?

**Pull request closed without merging**